### DISCUSSION TITLE
invoices for sales

### DISCUSSION THREAD

**Discussion by Marius9308** [gameserverapp/Platform#492] [Suggestions]
Would it be possible, to add a company data field with tax number and Adress, so that Supporters could get a invoice? Otherwise in some Countrys you will get trouble with the Tax Office and the law. 
Thank you!

**Comment by mvaessen** [comment]
Hi,

For reference, this was discussed on Discord: https://discord.com/channels/315049169071112193/945483534150303784/1135898516800151632

Your requests seems to involves:
- A way for your customers to download an invoice with the relevant information
- a way for players to add/update personal billing info (name, address etc)

**Reply by Marius9308** [reply]
Thats exactly what i was thinking about, yes.

**Reply by mvaessen** [reply]
For the meantime, have you considered adding the information to the footer of the site? I'm not 100% sure yet what kind of info you need to post exactly, but based on the information you mention on discord I assume this would solve the issue for now.

**Reply by Marius9308** [reply]
No, an imprint or footer is unfortunately not enough.
I know that in Germany we have very strict rules on this and that customs are very active, but I can't imagine that other countries don't have regulations along these lines.
But I'm really glad you're taking care of it. I think the alternative would be to shut down our cluster, since unfortunately there would always be the risk of high penalties.

**Reply by mvaessen** [reply]
Okido, can you please give a clear example of the kind of information that needs to be posted on the supporter tier page, and please also include a suggestions where this should be placed.

Based on other German sites, it seems that just the company info posted in the footer does suffice. This wont cover the invoice point though.

**Reply by Marius9308** [reply]
It would be quite sufficient if the players would get an invoice when buying a supporter tier with the content that was entered in advance in a ready-made text field.
e.g.
Purchased package
seller name
Seller Address
tax number
Convenient as a PayPal purpose.
However, we should be able to also download the sent invoice (in the best case via Gsa under the Sales tab)
That would be the simplest solution.
Everything else is possible via the footer and you don't need to change anything.
Thanks you for your time and sorry for the inconvenience!

**Reply by mvaessen** [reply]
We currently do not store information about the buyer, like their name or address.

There currently are no plans to store "personal info about buyers" in the near future, as this requires quite a lot of work.

**Reply by Marius9308** [reply]
No not the buyers, only the seller (me) is required

**Reply by mvaessen** [reply]
Ah, i misread. Sorry about that!

However, when we're adding new functionality I do prefer adding it in a way where it works for most customers. 
Since there are countries that do require the buyer info on the invoice I will need to add the buyer stuff anyway. As mentioned does this take more time to design and build, since we also need places for people to manage this info and such.

When the new docs are done I will have more time to look into this 👍

**Reply by Marius9308** [reply]
Thank you Max! 
That will safe the day for us.

**Comment by mvaessen** [comment]
Created development item #505

**Comment by mvaessen** [comment]
Hi,

Thanks again for your suggestion. This should now be available on your dashboard. For more information check out the docs: https://docs.gameserverapp.com/dashboard/monetization/getting_started#invoice-settings
